src/util/timeago.js

```diff
@@ -0,0 +1,57 @@
+TimeAgo = function TimeAgo() {
```

Comment:
This is really cool. Did you write this? You should consider releasing it as a stand-alone NPM module. It'd be useful and being maintainer on a small NPM module is a pretty interesting experience.

Reply:
@jywarren I didn't, it is from here: https://gist.github.com/caiotarifa/30ae974f2293c761f3139dd194abd9e5 but I had to modify it for what we needed!
I also used the modified version here: https://github.com/publiclab/plots2/blob/668c28e5822f17438b7b4da8d39e27d3af7f38a1/app/assets/javascripts/timeago.js

Reply:
Awesome. Would you have any time to try rebasing this to get it merged? I don't think it should be too tough... but i understand if you don't have time right now! Be safe and well!

Reply:
Noting that TimeAgo is not appearing on publiclab.org, i think we need to add a JS include there somewhere... https://publiclab.org/map#11/40.67759191045513/-73.87728530215101

https://github.com/publiclab/plots2/pull/7715/files#diff-a9c3bd311eab80c9ebe6a69830f9ad02R62 seems to include it though... hmm

Reply:
Sorry to clog up this thread. I think it's a load-order issue and i'll re-order it. All good!